### PR TITLE
Adds navigation bar

### PR DESCRIPTION
**Pull request by Kurocifer**
Close issue #9

### CODE REVIEW

**Comment by Kurocifer** [comment]
I don't know how bad this is. Please just give me feed back and I will see to improve it.

**Comment by warrenrhodes** [comment]
@Kurocifer please update you branch.

**Comment by Kurocifer** [comment]
Hey @warrenrhodes  please when you got time could I use your help. I checked the code you added in the themes.dart saw the section about the themes for the navigation bar but how could I use them for the different things like setting the colors of the lable and icon so it blends well when highlighted on dark theme as on light theme ?
[talking of this section](https://github.com/karibu-cap/sabitou-clients/blob/428a4d5bb686a5a7c827535ead6becf441abc310/sabitou_clients/lib/themes/themes.dart#L62-L76)

**Comment by warrenrhodes** [comment]
> Hey @warrenrhodes please when you got time could I use your help. I checked the code you added in the themes.dart saw the section about the themes for the navigation bar but how could I use them for the different things like setting the colors of the lable and icon so it blends well when highlighted on dark theme as on light mode ? [talking of this section you added](https://github.com/karibu-cap/sabitou-clients/blob/428a4d5bb686a5a7c827535ead6becf441abc310/sabitou_clients/lib/themes/themes.dart#L62-L76)

@Kurocifer https://rydmike.com/flexcolorscheme/themesplayground-latest/ here the are mentioned in the `icon avatar` section that the icon use the `foreground` color. now if you are not satisfy with the color, you can create a component inside the widget/component where the class that 2 parameter (label and iconData) and inside the class you check whether we are in dart mode (AppService.to.isDarkMode) then use Theme.of(context).colorschema.__

File: sabitou_clients/lib/widgets/admin_section_navigation_item.dart [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                Row(
+                  children: [
+                    const Padding(padding: EdgeInsets.only(left: 50)),
+                    Icon(
+                      navBarItem.icon,
+                      color: Colors.black54,
+                    ),
+                    const SizedBox(
+                      width: 10,
+                    ), // Separation between the icon and label.
```
Comment (warrenrhodes) [inline sabitou_clients/lib/widgets/admin_section_navigation_item.dart:64]: remove the comment here please

File: sabitou_clients/lib/widgets/admin_section_navigation_item.dart [hunk 2]
```diff
@@ -0,0 +1,103 @@
+import 'package:flutter/material.dart';
+
+import '../utils/constants.dart';
+import 'navigation_item.dart';
+
+/// Builds the admin navigation item and the items below it.
+class AdminSectionNavigationItem extends StatefulWidget {
```
Comment (warrenrhodes) [inline sabitou_clients/lib/widgets/admin_section_navigation_item.dart:7]: use the statelesswidget.

**Comment by warrenrhodes** [comment]
please generate the goldens for this view: here is the command flutter test --update-goldens --tags=golden file_name

**Comment by Kurocifer** [comment]
> please generate the goldens for this view: here is the command flutter test --update-goldens --tags=golden file_name

Updated the code as you asked Sir. And ran all the test that were defined by the scripts in the scripts directory.

**Pull request closed without merging**